Update Restart-VM.md (#2802)
MicrosoftDocs/windows-powershell-docs: docset/winserver2019-ps/hyper-v/Restart-VM.md

### -Timeout

> Specifies the duration of the wait, in seconds. When the timeout elapses, Restart-VM returns to the command prompt, even if the VM isn't  restarted.

Reviewer: ```suggestion
Specifies the duration of the wait, in seconds. When the timeout elapses, **Restart-VM** returns to the command prompt, even if the virtual machine isn't restarted.
```

> The Timeout parameter is only valid with the Wait parameter. Timeout overrides the Wait parameter's indefinite waiting period.

Reviewer: ```suggestion
This parameter is only valid with the `-Wait` parameter. The `-Timeout` parameter overrides the `-Wait` parameter's indefinite waiting period.
```

### -Wait

> Restart-VM suppresses the PowerShell prompt and blocks the pipeline until the VM has restarted. You can use this parameter in a script to restart VM and then continue to process when the restart is finished.

Reviewer: ```suggestion
**Restart-VM** suppresses the PowerShell prompt and blocks the pipeline until the virtual machine has restarted. You can use this parameter in a script to restart the virtual machine and then continue to process when the restart is finished.
```

> The Wait parameter waits indefinitely for the VM to restart. You can use Timeout to adjust the timing and the For and Delay parameters to wait for particular services to become available on the restarted VM.

Reviewer: ```suggestion
The `-Wait` parameter waits indefinitely for the virtual machine to restart. You can use the `-Timeout` parameter to adjust the timing and the `-For` and `-Delay` parameters to wait for particular services to become available on the restarted virtual machine.
```
